.github/workflows/continuous-integration.yml

```diff
@@ -526,7 +527,7 @@ jobs:
           repository: department-of-veterans-affairs/testing-tools-team-dashboard-data
           token: ${{ env.VA_VSP_BOT_GITHUB_TOKEN }}
           path: testing-tools-team-dashboard-data
-          ref: master
+          ref: Cbonade/refactor-classes
```

Comment:
I'm guessing this will eventually be changed back to `master`?

Reply:
yep just switched that right now to give it one more test run. This was on master when the prior approval came in